CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+- Added support for paid and authenticated relays (NIP-42).
```

Comment:
Do want to add NIP-42 to our README as well?

Reply:
💯